specification/netapp/resource-manager/readme.java.md

```diff
@@ -36,6 +37,19 @@ batch:
   - tag: package-2017-08-15
 ```
 
+### Tag: package-netapp-2021-02-01 and java
+
+These settings apply only when `--tag=package-netapp-2021-02-01 --java` is specified on the command line.
+Please also specify the `--azure-libraries-for-java-folder=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-netapp-2021-02-01' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.netapp.v2021_02_01
+  output-folder: $(azure-libraries-for-java-folder)/sdk/netapp/mgmt-v2021_02_01
+regenerate-manager: true
+generate-interface: true
+```
+
```

Comment:
@weidongxu-microsoft to confirm the change for Java readme

Reply:
It is fine, though not necessary. For T2 Java SDK build with swagger will follow the default tag in readme.md. In release request you can specify the tag, though we might not release old tag if current Java SDK is on new one.

Reply:
Thanks Weidong,  hi @audunn, let me know if you need to merge this PR

Reply:
> Thanks Weidong, hi @audunn, let me know if you need to merge this PR

Perhaps best to hold off for now as I hear deployments of this api-version are being rollined out and the api-version is yet not available in all locations.  Should be soon I hope and I will update here once they are out.